.proxy-version

```diff
@@ -1 +1 @@
-1c0d100
+9faa7ca
```

Comment:
can you list the proxy changes that this picks up in the description of this commit?

Reply:
When I squash this PR, I will list out the changes!

Reply:
I just updated the PR comment so that you can see what it will look like.